src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/Tensor.cs

```diff
@@ -691,6 +691,56 @@ private Tensor<T> GetTriangle(int offset, bool upper)
         /// <param name="value">The new value to set at the specified position in this Tensor.</param>
         public abstract void SetValue(int index, T value);
 
+        /// <summary>
+        /// The type that implements enumerators for <see cref="Tensor{T}"/> instances.
+        /// </summary>
+        public struct Enumerator : IEnumerator<T>
+        {
+            private readonly Tensor<T> tensor;
+            private int index;
+
+            internal Enumerator(Tensor<T> tensor)
+            {
+                Debug.Assert(tensor != null);
+
+                this.tensor = tensor;
```

Comment:
```suggestion
                _tensor = tensor;
```
See https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md